apps/cowswap-frontend/src/modules/trade/containers/TradeWidget/TradeWidgetForm.tsx

```diff
@@ -26,6 +32,13 @@ import { useIsWrapOrUnwrap } from '../../hooks/useIsWrapOrUnwrap'
 import { TradeWidgetLinks } from '../TradeWidgetLinks'
 import { WrapFlowActionButton } from '../WrapFlowActionButton'
 
+const scrollToMyOrders = () => {
+  const element = document.getElementById(MY_ORDERS_ID)
```

Comment:
I think it's ok to use `"my-orders"` here instead of const, otherwise we should use the const in `OrdersTableContainer/index.tsx` as well

Reply:
Addressed this by also using the `const` in the other file